app/hooks/useModals.tsx

```diff
@@ -1,41 +1,37 @@
 import { useMantineTheme } from "@mantine/core"
-import {
-  openConfirmModal,
-  openModal,
-  useModals as MantineUseModals,
-} from "@mantine/modals"
+import { modals, useModals as useMantineModals } from "@mantine/modals"
 import { ModalSettings, OpenConfirmModal } from "@mantine/modals/lib/context"
 
 const useModals = () => {
   const theme = useMantineTheme()
-
-  const { modals } = MantineUseModals()
+  const { modals: openModals } = useMantineModals()
 
   const commonModalProps: ModalSettings = {
     centered: true,
-    overlayColor:
-      theme.colorScheme === "dark" ? theme.colors.navy[9] : theme.colors.gray[2],
-    overlayOpacity: 0.8,
-    overlayBlur: 3,
+    overlayProps: {
+      color: theme.colorScheme === "dark" ? theme.colors.navy[9] : theme.colors.gray[2],
+      opacity: 0.8,
+      blur: 3,
+    },
     padding: "md",
   }
 
   const openConfirmationModal = (modalProps: OpenConfirmModal) =>
```

Comment:
These functions are nice. If you wanted, I think you could go one step further and create an openModal function which takes a "type" prop. This would simplify the useModals functionality and make the pattern for extending it easier.

ex:
`
// usage
openModal("confirm")

// function
const openModal = (type: string, modalProps: ModalSettings) => {
  switch(type) {
    case "confirm":
      openConfirmationModal(modalProps)
      break;
    ...
  }
}
`